src/mrsimulator/tests/test_core.py

```diff
@@ -103,7 +111,11 @@ def test_parse_json_isotopomer():
 
 
 def test_direct_init_spectrum():
-    Spectrum()
+    the_spectrum = Spectrum()
+    assert the_spectrum.number_of_points == 1024
+    assert the_spectrum.spectral_width == 100
+    # assert the_spectrum.property_units["spectral_width"] == 'Hz'
```

Comment:
Same as with the Site class, the units are undefined when the instance of the Spectrum class is defined directly.

Reply:
The units issue is something I'm still thinking of how to get around. Supplying strings in python for floats is very un-pythonic. I'm thinking I might make the units a separate variable and enforce that all the units are the same, IE all ppm or all Hz within a structure.